### ISSUE TITLE
[Accessibility]:  Button "Show more" does not announce a change in the view

### ISSUE THREAD

**Issue by Numbers88s** [sourcegraph/sourcegraph#35250]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33506

### Problem description

<img width="1648" alt="Screen Shot 2022-05-10 at 12 13 51 PM" src="https://user-images.githubusercontent.com/6225160/167675010-5daa167e-7a1a-42e0-bf98-dc91c7002926.png">

Clicking "Show more" does not announce a change in the view (it should say "200+ total results" or "100 more results loaded")

### Expected behavior

I would like to hear how much the results changed after clicking the "Show More" button.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by macraig** [comment]
Closing as identified issues have been completed
